### PR TITLE
feat: update navbar

### LINKED ISSUE #87: [Task]: Update navbar to new design system
**Task description.**

Update the navbar component to use Primer CSS and follow the figma design https://www.figma.com/file/Fe2hyux4KGmKjeni2nb2mg/Sugar-Rush?node-id=62%3A7875. Only what is stated in requirements needs to be done, there are other addittions to the navbar that will be added later

image

**Requirements.**
 - Logo and project name
 - Wallet connection

### PR DESCRIPTION
**Pull request by evavirseda**
Fixes #87 

## Changes

  - Update the navbar component to use Primer CSS
  - Add Logo and wallet connection
  
### Checks

* [x] Have you catchup your branch with the latest state in base?
* [x] Have you lint your code locally prior to submission?
* [x] Have you reviewed your proposed changes and removed debris?
* [x] Have you successfully ran due tests with your changes?

### CODE REVIEW

**Comment by evavirseda** [comment]
The previous design also included tooltips for navbar items, but it may not be necessary now, what do you think? @VmMad @cpl121

**Comment by VmMad** [comment]
Yeah I don't think any tooltips are necessary here. The ones in the PR right now are kinda redundant

File: components/Navbar.tsx [hunk 1]
```diff
@@ -10,77 +12,100 @@ interface INavbarElement {
...
+                    </div>
+                    {open && (
+                        <NavList
+                            className='d-flex position-absolute top-8 color-bg-emphasis color-fg-subtle width-full left-0'
+                            sx={{ height: '100vh' }}
+                        >
+                            {NAVBAR_ELEMENTS.map((element) => {
+                                return (
+                                    <Link href={element.url} key={element.url}>
+                                        <NavList.Item
```
Comment (VmMad) [inline components/Navbar.tsx:90]: Could you add an onClick close navbar on mobile?

**Review by VmMad — APPROVED** [review]
Looks good :+1:

File: components/Navbar.tsx [hunk 2]
```diff
@@ -1,86 +1,104 @@
...
 const Navbar: FC = () => {
-    const { publicKey } = useWallet()
-
+    const router = useRouter()
+    const [open, setOpen] = useState(false)
     const NAVBAR_ELEMENTS: INavbarElement[] = [
         {
-            title: 'Manage CMs',
+            title: 'Dashboard',
             url: '/',
```
Comment (cpl121) [inline components/Navbar.tsx:15]: The path of `/` should be `Dashboard/Candy Machines`

File: components/Navbar.tsx [hunk 3]
```diff
@@ -1,86 +1,104 @@
...
-            tooltip: 'Manage Candy Machines',
         },
         {
-            title: 'Create CMs',
+            title: 'Candy Machines',
             url: '/create-candy-machine',
```
Comment (cpl121) [inline components/Navbar.tsx:19]: `Create candy machine` is a modal component

File: styles/globals.scss [hunk 4]
```diff
@@ -30,15 +30,25 @@ h1 {
...
+h6 {
     @apply text-lg;
 }
 
 a {
     text-decoration: none;
     @apply text-blue-regular;
 }
+
+.wallet-adapter-button {
```
Comment (cpl121) [inline styles/globals.scss:48]: Text size could be a bit smaller

File: components/Navbar.tsx (hunk 2 above)
Comment (evavirseda) [reply components/Navbar.tsx]: you mean the title right?

**Review by cpl121 — APPROVED** [review]
Nice job!

**Pull request merged**